static/js/pm_conversations.js

```diff
@@ -1,16 +1,15 @@
-"use strict";
-
-const FoldDict = require("./fold_dict").FoldDict;
+import {FoldDict} from "./fold_dict.ts";
```

Comment:
Don’t add the `.ts` extension to imports—TypeScript will actually reject this unconditionally (TS2691).

Reply:
Fixed by adding aee575fcbbac2e2d87406ebb01fed011d84c61a3 and then removing `.ts` from import. `eslint-plugin-import` was reporting and error so I had to add it. (I had this in my list of fix but lost track of it when making a PR!)